src/docs/sphinx/QuickStart.rst

```diff
@@ -237,120 +233,107 @@ If something goes wrong, the first thing the support team will ask you for is th
 
 .. note::
    If you develop a new ``host-config`` for a particular platform that may be useful for other users, please consider sharing it with the developer team.
 
 Compilation
-==================
+===========
+
+The configuration process for both the third-party libraries (TPLs) and GEOS is managed through a Python script called ``config-build.py``. This script simplifies and automates the setup by configuring the build ad install directories and by running CMake based on the options set in the host-config file 
+which is passed as a command-lne argument. The ``config-build.py`` script has several command-line options. Here, we will only use some basic options and rely on default values for many others. During this build process there wil be automatically generated build and install directories for both the TPLs and the main code,
+with names consistent with the name specified in the host-config by the variable ``CONFIG_NAME``, i.e. ``build-your-platform-release`` and ``install-your-platform-release``. 
+
+All options can be visualized by running
+
+.. code-block:: sh
+
+   cd thirdPartyLibs
+   python scripts/config-build.py -h
+
+.. note::
+
+   It is strongly recommended that GEOS and TPLs be configured using the same host configuration file. Below, we assume that you keep this file in, for example, ``GEOS/host-configs/your-platform.cmake``, but the exact location is up to you.
 
-We will begin by compiling the TPLs, followed by the main code.
-If you work on an HPC system with other GEOS developers, check with them to see if the TPLs have already been compiled in a shared directory.
-If this is the case, you can skip ahead to just compiling the main code.
-If you are working on your own machine, you will need to compile both.
+Compiling the TPLs
+-------------------
 
-We strongly suggest that GEOS and TPLs be built with the same hostconfig file.
-Below, we assume that you keep it in, say, ``GEOS/host-configs/your-platform.cmake``, but this is up to you.
+.. note::
 
-We begin with the third-party libraries, and use a python ``config-build.py`` script to configure and build all of the TPLs.
-Note that we will request a Release build type, which will enable various optimizations.
-The other option is a Debug build, which allows for debugging but will be much slower in production mode.
-The TPLS will then be built in a build directory named consistently with your hostconfig file.
+   If you are working on an HPC system with other GEOS developers, check with them to see if the TPLs have already been compiled in a shared directory. If this is the case, you can skip ahead to just compiling the main code.
+   If you are working on your own machine, you will need to configure and compile both the TPLs and the main code.
+
+We begin by configuring the third-party libraries (TPLs) using the ``config-build.py`` script. This script sets up the build directory and runs CMake to generate the necessary build files.
```

Comment:
In the `quick-start-template.cmake`, it is currently configured that:

```
# TPLs
set( ENABLE_TRILINOS OFF CACHE PATH "" FORCE )
set( ENABLE_CALIPER OFF CACHE PATH "" FORCE )
set( ENABLE_DOXYGEN OFF CACHE BOOL "" FORCE)
set( ENABLE_MATHPRESSO OFF CACHE BOOL "" FORCE )
```

Still, these libraries are built (without needed). What I meant in my comment was to not ask the users to build these, to save time and maybe reduce the possibility of things going wrong.

Reply:
I have these same settings in the `host-config` I use to build on my local machine and these libraries are not built. It's what we also do for the macOS [ host config ](https://github.com/GEOS-DEV/GEOS/blob/develop/host-configs/apple/macOS_base.cmake)

Did you try to rerun the config script for the tpls using this host config (or adding these lines to the one you were using) and observed that the libraries were still built?

Reply:
You are right, I did not notice that the host config in this PR was different in this aspect than the one I used before. False alarm.